tests/installation/partitioning_raid.pm

```diff
@@ -316,8 +316,8 @@ sub add_prep_boot_partition {
     assert_screen 'partitioning_raid-format_noformat';
     send_key 'alt-i';
     assert_screen 'partitioning_raid-file_system_id-selected';
-    my $direction_key = (is_storage_ng) ? 'up' : 'down';
-    send_key_until_needlematch 'filesystem-prep', $direction_key;
+    send_key 'up' for (1 .. 8);
```

Comment:
I don't like magic numbers as we will break tests in case there are 9 items in the list. Have you tried `home` button? I think it should work. I believe, I've introduced that code, so have to admit that have used cheapest solution =).

Reply:
In case it's urgent, let's bump number of key presses to 15, which should be safe for some years and merge =).

Reply:
Thanks, will try home this afternoon :)